src/components/UIShell/UIShell.svelte

```diff
@@ -14,20 +14,24 @@
   import UIShellSideNavWrapper from './UIShellSideNav/UIShellSideNavWrapper.svelte';
   import UIShellSideNavItem from './UIShellSideNav/UIShellSideNavItem.svelte';
   import HamburgerMenu from './UIShellSideNav/HamburgerMenu.svelte';
-
+  import { onMount } from 'svelte' 
+  
   let isSideNavOpen = undefined;
-  let winWidth = window.innerWidth;
-
-  window.addEventListener('resize', () => {
-    winWidth = window.innerWidth;
-
-    if (winWidth >= 1056) {
-      isSideNavOpen = true;
-    } else {
-      isSideNavOpen = false;
-    }
-  });
+  let winWidth = undefined;
+  
+  onMount(() => {
+    winWidth = window.innerWidth
+    window.addEventListener('resize', () => {
```

Comment:
Using the `svelte:window` API will automatically remove the event listener when destroyed.

```
<svelte:window on:resize="..." />
```

Reply:
Thank you, I changed the code to use <svelte:window />; see [revision](https://github.com/IBM/carbon-components-svelte/pull/149/commits/ee27d7b8cd9d12688a173f2c4378bf75bf841a90)

Appreciate the help